### DISCUSSION TITLE
The crystal structure become disorder abruptly in MD simulation

### DISCUSSION THREAD

**Discussion by StealFox97** [deepmodeling/deepmd-kit#2708] [Q&A]
Dear developers,
I trained a cubic perovskite model to study the influence of doping elements. All training sets were generated by CP2K2023. Two different doped perovskite models with 3*3*3 unit cell were considered. The AIMD simulations were used to generate the data set, using the NVT ensemble at 1000K in 1 ps (Timestep is 1 fs). After that, I trained 1000000 steps with deepmd-kit and got the graph-compress.pb file.
But when I run MD simulation with Lammps package, I met a serious issue. **The crystal structure become disorder abruptly at a certain simulation time.** I tried different amounts of doping atoms, the concentration of oxygen vacancies and simulation temperatures. Even the pure perovskite model without oxygen vacancies at 500 K, the crystal structure become disorder also.
<img width="1083" alt="突然无序化" src="https://github.com/deepmodeling/deepmd-kit/assets/139855013/0f1d2aeb-fc23-49fd-a383-2a542c0da450">
I know it is the potential model problem. But I don't know how to solve it. Should I consider the perfector of virial loss when I train the model. I notice that its default value is zero. Or should I think about what I did mistake elsewhere.
Someone can help me?
In addition, I have some confused when using deepmd-kit: 
1.	The DP models is only a function of atomic coordinates, does it mean that I don’t need to enter the information of bond and angle in Lammps data files, even I want to simulation some molecular system?
2.	If I want to study the solid surface, should I prepare the bulk data sets, gas molecular data sets and slab data sets respectively? 
The "input.json" file used is as below:
```json
{
    "_comment": " model parameters",
    "model": {
	"type_map":	["Fe", "O", "Ca", "Ba", "H"],
	"descriptor" :{
	    "type":		"se_e2_a",
	    "sel":		"auto",
	    "rcut_smth":	0.50,
	    "rcut":		6.00,
	    "neuron":		[25, 50, 100],
	    "resnet_dt":	false,
	    "axis_neuron":	16,
	    "seed":		1,
	    "_comment":		" that's all"
	},
	"fitting_net" : {
	    "neuron":		[240, 240, 240],
	    "resnet_dt":	true,
	    "seed":		1,
	    "_comment":		" that's all"
	},
	"_comment":	" that's all"
    },

    "learning_rate" :{
	"type":		"exp",
	"decay_steps":	5000,
	"start_lr":	0.001,	
	"stop_lr":	3.51e-8,
	"_comment":	"that's all"
    },

    "loss" :{
	"type":		"ener",
	"start_pref_e":	0.02,
	"limit_pref_e":	1,
	"start_pref_f":	1000,
	"limit_pref_f":	1,
	"start_pref_v":	0,
	"limit_pref_v":	0,
	"_comment":	" that's all"
    },

    "training" : {
	"training_data": {
	    "systems":		["../training_data/data_0/", "../training_data/data_1/", "../training_data/data_2/"],
	    "batch_size":	"auto",
	    "_comment":		"that's all"
	},
	"validation_data":{
	    "systems":		["../validation_data/data_0/", "../validation_data/data_0/"],
	    "batch_size":	1,
	    "numb_btch":	1,
	    "_comment":		"that's all"
	},
	"numb_steps":	1000000,
	"seed":		10,
	"disp_file":	"lcurve.out",
	"disp_freq":	1000,
	"save_freq":	2000,
	"_comment":	"that's all"
    },    

    "_comment":		"that's all"
}
```

**Comment by wanghan-iapcm** [comment]
I am afraid the 1ps AIMD simulation would not give a good sampling of the configuration space, therefore, the model could not correctly predict the energy and force at some off-equilibrium configuration that may lead to the disordered structures. 

You may want to try the [dpgen](https://github.com/deepmodeling/dpgen) scheme that automatically generate the ensemble of training data from which the model is trained to a uniform accuracy on the interested configuration space. 

You may also want to try the [DPA-1 model](https://docs.deepmodeling.com/projects/deepmd/en/master/model/train-se-atten.html) which significantly outperforms the deeppot-se (`se_a` descriptor).
